crypto/asn1/a_utf8.c

```diff
@@ -151,7 +117,7 @@ int UTF8_putc(unsigned char *str, int len, unsigned long value)
         }
         return 3;
     }
-    if (value < 0x200000) {
+    if (value < 0x110000) {
```

Comment:
The comparison is against `0x10ffff` in `crypto/asn1/a_mbstr.c `.  I think it would be better if these were consistent i.e. both `0x10ffff` or `0x110000`.  I don't have a preference for either.

Reply:
Done.